asv_bench/benchmarks/stat_ops.py

```diff
@@ -1,205 +1,122 @@
-from .pandas_vb_common import *
+import numpy as np
+import pandas as pd
 
+from .pandas_vb_common import setup  # noqa
 
-def _set_use_bottleneck_False():
-    try:
-        pd.options.compute.use_bottleneck = False
-    except:
-        from pandas.core import nanops
-        nanops._USE_BOTTLENECK = False
 
+class Bottleneck(object):
 
-class FrameOps(object):
     goal_time = 0.2
+    params = ([True, False], ['DataFrame', 'Series'])
+    param_names = ['use_bottleneck', 'constructor']
 
-    param_names = ['op', 'use_bottleneck', 'dtype', 'axis']
-    params = [['mean', 'sum', 'median'],
-              [True, False],
-              ['float', 'int'],
-              [0, 1]]
-
-    def setup(self, op, use_bottleneck, dtype, axis):
-        if dtype == 'float':
-            self.df = DataFrame(np.random.randn(100000, 4))
-        elif dtype == 'int':
-            self.df = DataFrame(np.random.randint(1000, size=(100000, 4)))
-
-        if not use_bottleneck:
-            _set_use_bottleneck_False()
+    def setup(self, use_bottleneck, constructor):
+        values = np.random.randn(10**6)
+        self.data = getattr(pd, constructor)(values)
+        try:
+            pd.options.compute.use_bottleneck = use_bottleneck
+        except:
+            from pandas.core import nanops
+            nanops._USE_BOTTLENECK = use_bottleneck
 
-        self.func = getattr(self.df, op)
+    def time_mean(self, use_bottleneck, constructor):
+        self.data.mean()
 
-    def time_op(self, op, use_bottleneck, dtype, axis):
-        self.func(axis=axis)
 
+class FrameOps(object):
 
-class stat_ops_level_frame_sum(object):
     goal_time = 0.2
+    param_names = ['op', 'dtype', 'axis']
+    params = [['mean', 'sum', 'median', 'std'],
```

Comment:
and use bottleneck here, also expand these to all of the stat ops (min, max, var, kurt, etc)